### ISSUE TITLE
Write a blog that Yoruba is now available on Easy Worship

### ISSUE THREAD

**Issue by emmanlex** [DaBible-Foundation/DaBible-MainHub#45]
Write a blog that Yoruba is now available on Easy Worship using a minimum of 250 words

**Comment by emmanlex** [comment]
{H1} - Yoruba Bible is now available on EasyWorship
{Image}
Good news to all our readers and app users DaBible Foundation Yoruba Bible Version is now available on EasyWorship. For many of you who do not know what easyworship is and are unable to spend the data to go look up that link (I understand you perfectly). In simple words only, EasyWorship is a software used for displaying bible texts and presenting in many churches. If your church has a TV, project or screen that displays bible verses while someone is preaching or singing, then most likely your church is using easyworship. EasyWorship is a faith-based software. Faith-based meaning it is intended for believers and to be used in church settings. EasyWorship is used by many churches around the world for their services. That is why DaBible Foundation is proud to have our very own Yoruba Bible version on EasyWorship. DaBible Foundation, the NGO (non-governmental organization) that developed the Free Yoruba Audio Bible app (YAB) and Free Pidgin Audio Bible app (PAB) now has its Yoruba Bible Version on EasyWorship. Imagine reading the Yoruba bible on your large screens, projectors or LCD walls. Wow. Personally, it is a dream come true. Likewise, at DaBible Foundation we believe this is a step forward in the right direction. Our mission will always be to share the gospel of peace. 

In our previous blog post, Vote and encourage Yoruba Bible on EasyWorship, we informed our readers that we put in a request to have the Yoruba Bible Version on EasyWorship. Not long after, the EasyWorship support team responded back to us and thankfully it was a favorable response. So, we began talks on the project and three months later we are here to announce the completion of this project 

{H2} - Steps to using Yoruba Bible on Easyworship
Now the option is available. 

Step One 

Go to your easyworship software on your pc (not available on mac sadly) open the software and then head to the Scripture icon on the left side of your easyworship software. 

On the the Scripture section, click “More available”  the 

@sanmiayotunde will continue to work on it the soonest
